### PR TITLE
[SPIRV] Do not include input file in DebugEntryPoint args

### PR DESCRIPTION
**Pull request by greg-lunarg**

### CODE REVIEW

**Comment by greg-lunarg** [comment]
@Keenuts or @cassiebeckley Please review when CI clears. Thanks!

**Review by Keenuts — COMMENTED** [review]
LGTM, but there are changes to DXC, which I don't know if I can approve.

**Comment by cassiebeckley** [comment]
It looks like all the changes are to code for the SPIR-V (or options for the SPIR-V backend), so I think it should be fine for us to approve

File: tools/clang/tools/dxcompiler/dxcompilerobj.cpp [hunk 1]
```diff
@@ -1019,10 +1019,16 @@ class DxcCompiler : public IDxcCompiler3,
...
-        // Store a string representation of command line options.
-        if (opts.DebugInfo)
-          for (auto opt : mainArgs.getArrayRef())
-            opts.SpirvOptions.clOptions += " " + std::string(opt);
+        // Save a string representation of command line options and
+        // input file name.
+        if (opts.DebugInfo) {
+          opts.SpirvOptions.inputFile = opts.InputFile;
+          for (auto opt : mainArgs.getArrayRef()) {
+            if (opts.InputFile.compare(opt) != 0) {
```
[the remark this answers is not in the document]
Comment (greg-lunarg) [reply tools/clang/tools/dxcompiler/dxcompilerobj.cpp:1027]: Not sure. I was just copying prior art.

**Comment by greg-lunarg** [comment]
@Keenuts or @cassiebeckley Thank you for the review and approval. Could someone please merge? I don't have authorization.

**Pull request merged**